qi-lib/flow/core/normalize.rkt

```diff
@@ -4,84 +4,85 @@
 
 (require (for-syntax racket/base
                      syntax/parse
-                     "pass.rkt"
+                     "strategy.rkt"
                      "private/form-property.rkt")
          "passes.rkt")
 
 ;; 0. "Qi-normal form"
 (begin-for-syntax
+  (define (normalize-rewrite stx)
```

Comment:
These `define-for-syntax`-type functions are almost begging to be written in a module imported at the right phase level, but I recall there were some technical challenges. 😢

Reply:
Last time when we discussed this, we found there were some cases where there are both syntax phase and runtime phase definitions, and one option was to have two separate modules containing these things (e.g. `(require deforestation-runtime)` to simply how they are defined.

But yes, mainly this is still something we need to work out!